docs/admin/deploy/docker-compose/index.mdx

```diff
@@ -116,8 +116,9 @@ Continue with the following steps *after* you have created a public or private c
   - It also allows us to track all of the customizations made to your Sourcegraph instance.
 
 ```bash
-  # Specify the version you want to install
-  export SOURCEGRAPH_VERSION="v5.3.9104"
+  # Specify the version you want to install - find the latest version at sourcegraph.com/changelog
```

Comment:
This should probably point to our [Releases](https://sourcegraph.com/docs/releases) page instead of the changelog.